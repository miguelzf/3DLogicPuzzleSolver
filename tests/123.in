- - - G -
- P - - -
- - - - -
- - - - -
- - - - -

- - - - B
- - - L -
- - - R P
- - - - -
- - - - -

G R - L -
- - - - -
- - - - -
- - - B -
- - - - -
