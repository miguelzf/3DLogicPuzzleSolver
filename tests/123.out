
Solution:
r  r  r  G  g  
r  P  r  l  l  
r  p  r  l  g  
r  p  r  l  g  
r  p  r  l  g  
=============
r  p  r  l  B  ||  G  R  g  L  g  
r  p  r  L  b  ||  b  r  g  g  g  
r  p  r  R  P  ||  b  r  r  r  r  
r  p  p  p  p  ||  b  b  b  B  r  
r  r  r  r  r  ||  r  r  r  r  r  
